pkg/routes/health.go

```diff
@@ -0,0 +1,54 @@
+package routes
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"time"
+
+	"github.com/blockchain-abstraction-middleware/project-template/pkg/server"
+	"github.com/go-chi/chi"
+)
+
+// HealthResource implements server.Resource interface
+type HealthResource struct {
+	path string
+}
+
+// NewResource constructor func
+func (r *HealthResource) NewResource(path string) server.Resource {
```

Comment:
I know this is my code lol but originally I had the `HealthResource` as `healthResource` (private) but I was gonna suggest changing it to being like this. Then we can remove the `server.Resource` return type and just return a `*HealthResource` and it'll work exactly the same as it implicitly implements the interface and then the pkg has no dependency on `server` pkg

Reply:
Ace => updated here in latest branch!